### PR TITLE
Fix/Admin V3 - Create Spaces bug fixes

### PR DESCRIPTION
**Pull request by yurii-lubynets**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1177

Before
Screenshot 2021-09-24 at 10 34 53
After
Screenshot 2021-09-24 at 10 40 35

### CODE REVIEW

File: src/pages/Admin/Details/Form/DetailsForm.styles.ts
Comment (goran-peoski-work) [inline src/pages/Admin/Details/Form/DetailsForm.styles.ts:16]: Please don't add more styled components, we're in the process of replacing them with SCSS+BEM per component

Comment (goran-peoski-work) [inline src/pages/Admin/Details/Form/DetailsForm.styles.ts:16]: Please don't add more styled components, we're in the process of replacing them with SCSS+BEM per component

File: src/pages/Admin/Details/Form/DetailsForm.tsx
Comment (goran-peoski-work) [inline src/pages/Admin/Details/Form/DetailsForm.tsx:216]: Can just as easily add `<Form className="">` and deal with it in an SCSS file

File: src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx
Comment (goran-peoski-work) [inline src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:40]: Why remove `AdminRestricted`?
Only people with admin rights should be able to create venues.

Comment (goran-peoski-work) [reply src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:40]: In order to update styles, a simple `VenueWizardCreate.scss` can have

```scss

.VenueWizardCreate {
  .AdminRestricted {
    // scrollbar fix
  }
}

```

This is the flexibility provided by diligently setting up block classes on all root elements of all components.

File: src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx
Comment (yurii-lubynets) [inline src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:46]: `<AdminRestricted/>` was deleted due to there are nested components
Screenshot 2021-09-23 at 17 03 48

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Changes requested.

Styled components easily replaceable by SCSS+BEM
Losing admin restriction functionality because of a style is unwarranted

File: src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx
Comment (goran-peoski-work) [reply src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:46]: `AdminRestricted`'s primary role is not visual. Styles can be updated without losing the restriction provided by it.

File: src/pages/Admin/Details/Form/DetailsForm.styles.ts
Comment (yurii-lubynets) [reply src/pages/Admin/Details/Form/DetailsForm.styles.ts]: removed in d205d698e7827ce16d8dfff434b02f7df4ff5fd1

File: src/pages/Admin/Details/Form/DetailsForm.tsx
Comment (yurii-lubynets) [reply src/pages/Admin/Details/Form/DetailsForm.tsx]: fixed in d205d698e7827ce16d8dfff434b02f7df4ff5fd1

File: src/pages/Admin/Details/Form/DetailsForm.tsx
Comment (goran-peoski-work) [inline src/pages/Admin/Details/Form/DetailsForm.tsx:253]: Can also add:
```suggestion
          loading={isSubmitting}
        >
```

File: src/pages/Admin/Details/Form/DetailsForm.tsx
Comment (goran-peoski-work) [inline src/pages/Admin/Details/Form/DetailsForm.tsx:224]: [OOS]

If there aren't that many of these `S.` components, it might be good opportunity to just move those styles in `DetailsForm.scss`

File: src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx
Comment (goran-peoski-work) [inline src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx]: Suggesting we leave this component in place. Remove the secondary (visual) effect of `AdminRestricted` itself as per https://doesitsparkle.slack.com/archives/C028JG5TUMC/p1632471222090800?thread_ts=1632466598.085100&cid=C028JG5TUMC

**Review by goran-peoski-work — APPROVED** [review]
Approving with a do-not-merge until `AdminRestricted` is resolved in next PR

**Comment by goran-peoski-work** [comment]
Soft do-not-merge. No harm if merged. Just the issue with the scrollbars and/or duplicate nested `AdminRestricted` can be done prior to this. Decision is up to @yurii-lubynets @whoaitsaimz timetable.

File: src/pages/Admin/Details/Form/DetailsForm.tsx
Comment (yurii-lubynets) [reply src/pages/Admin/Details/Form/DetailsForm.tsx]: fixed in b30532c2bb1f03501255b92b38a51d82471e0261

**Comment by yurii-lubynets** [comment]
Create space/Edit Space forms are fixed. Only one way I've found is to add extra space for the scroll on the right using 
@goran-peoski-work's hack https://github.com/sparkletown/sparkle/pull/2341/files#diff-699898e6115b7662e6eddb7c7073be6b4b73e038b63d93bd80f249812e037110R15

So we don't need to update the `AdminRestricted` component
Screenshot 2021-09-24 at 12 52 32


Screenshot 2021-09-24 at 12 52 44

File: src/components/atoms/AdminRestricted/AdminRestricted.scss
Comment (goran-peoski-work) [inline src/components/atoms/AdminRestricted/AdminRestricted.scss:15]: Allowing it for now, hopefully deleted soon with PR https://github.com/sparkletown/sparkle/pull/2345

**Review by goran-peoski-work — APPROVED** [review]
LGTM

**Comment by goran-peoski-work** [comment]
Related PR https://github.com/sparkletown/sparkle/pull/2345

**Pull request merged**
